youtube_dl/extractor/niconico.py

```diff
@@ -40,7 +40,7 @@ class NiconicoIE(InfoExtractor):
             'uploader': 'takuya0301',
             'uploader_id': '2698420',
             'upload_date': '20131123',
-            'timestamp': 1385182762,
+            'timestamp': int, # timestamp is unstable
```

Comment:
>- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

Reply:
Sorry, forgot to ran it again after the changes.